packages/debugger/test/data/helpers.js

```diff
@@ -4,7 +4,6 @@ const debug = debugModule("test:data:decode");
 import Ganache from "ganache-core";
 import { assert } from "chai";
 import changeCase from "change-case";
-import flatten from "lodash.flatten";
```

Comment:
Have we stopped using this now? Does this mean it's no longer necessary as a dependency? (I haven't looked)

Reply:
No, it's still used elsewhere.  We could get rid of it and just use `[].concat`, but, whatever.  (I also *added* the use of `lodash.merge`, which is harder to replace...)